common/io/Serial.cpp

```diff
@@ -157,11 +154,16 @@ bool OpenAndFlock(const std::string &path, int oflag, int *fd) {
     return false;
   }
 
+#ifdef HAVE_FLOCK
   if (flock(*fd, LOCK_EX | LOCK_NB) == -1) {
     OLA_INFO << "Failed to flock() device " << path;
     close(*fd);
     return false;
   }
+#else  // HAVE_FLOCK
+  close(*fd);
```

Comment:
Probably chuck a log line in here saying it's tried to flock without flock, so someone doesn't get confused why it never locks if this branch is ever triggered.

Reply:
Ok, done.  Now it will OLA_WARN "Tried to lock XXX, but flock() is unavailable.".